### PR TITLE
Update descriptives.b.R

### PR DESCRIPTION
**Pull request by atpinto**
Changed the critical value for the confidence interval for the mean. The critical values is now chosen as the quantile from the t-distribution with df=n-1 rather than the quantile from the normal distribution.

### CODE REVIEW

**Comment by atpinto** [comment]
The confidence interval for the mean in the Descriptive statistics command is using the critical values from the normal distribution instead to the t-distribution.  When the sample size is small size, this confidence interval will fail to have the correct coverage because is too narrow.   Given that we are using the sample standard deviation for the calculation of the standard error, the t-distribution should be used for the calculation of the confidence interval for the mean.

The correct confidence interval can be obtained form the t-test command and it will not match the normal-based  confidence interval in the descriptives command.

File: R/descriptives.b.R [hunk 1]
```diff
@@ -1417,8 +1417,10 @@ descriptivesClass <- R6::R6Class(
                 stats[['min']] <- min(column)
                 stats[['max']] <- max(column)
                 stats[['se']] <- sqrt(var(column)/length(column))
-                zQuant <- 1 - ((1 - self$options$ciWidth/100) / 2)
-                ciDiff <- qnorm(zQuant) * stats[['se']]
+                #zQuant <- 1 - ((1 - self$options$ciWidth/100) / 2)
```
Comment (raviselker) [inline R/descriptives.b.R:1420]: Please remove commented out line
```suggestion
```

File: R/descriptives.b.R [hunk 2]
```diff
@@ -1417,8 +1417,10 @@ descriptivesClass <- R6::R6Class(
...
+                tQuant <- 1 - ((1 - self$options$ciWidth/100) / 2)
+                #ciDiff <- qnorm(zQuant) * stats[['se']]
```
Comment (raviselker) [inline R/descriptives.b.R:1422]: Please remove commented out line
```suggestion
```

File: R/descriptives.b.R [hunk 3]
```diff
@@ -1417,8 +1417,10 @@ descriptivesClass <- R6::R6Class(
...
+                ciDiff <- qnorm(tQuant, df=length(column)-1 ) * stats[['se']]
```
Comment (raviselker) [inline R/descriptives.b.R:1423]: Improve formatting
```suggestion
                ciDiff <- qnorm(tQuant, df=length(column) - 1) * stats[['se']]
```

**Review by raviselker — CHANGES_REQUESTED** [review]
Sorry for the delay in the review, we were discussing internally for a bit wether we wanted accept this change and we came to the conclusion that it would indeed be better to use a t-distribution. So after the requested changes are applied, I'll approve this PR.

**Comment by jonathon-love** [comment]
i think we want to add a footnote too @raviselker ? 1. so people are aware of the change. 2. because it's informative to have?

**Comment by raviselker** [comment]
I fixed up this PR in #348 (there were some other issues with it).

**Pull request closed without merging**